jupyter-extension/src/plugins/mount/customFileBrowser.ts

```diff
@@ -85,34 +87,46 @@ const createCustomFileBrowser = (
       const commands = new CommandRegistry();
       commands.addCommand('file-open', {
         label: 'Open',
-        icon: 'fa fa-folder',
+        icon: new LabIcon({
+          name: 'folder-icon',
+          svgstr:
+            '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--!Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.--><path d="M464 128H272l-64-64H48C21.5 64 0 85.5 0 112v288c0 26.5 21.5 48 48 48h416c26.5 0 48-21.5 48-48V176c0-26.5-21.5-48-48-48z"/></svg>',
```

Comment:
Is this something that should be living in the `svg-images` directory? I don't think it's matters all that much but I'm guessing the svgs are intended for there?

Reply:
Yea, that would probably be better. However, I realized while cleaning this up that we can just use the icons defined in jupyter's `ui-components` which has the added benefit of a more cohesive feel